.github/workflows/publish.yml

```diff
@@ -1,38 +0,0 @@
-name: publish_docs
-
-on:
-  push:
-    branches:
-    - main
-    paths:
-    - 'docs/sources/**'
-
-jobs:
-  # test:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - name: Check out code
-  #       uses: actions/checkout@v3
-  #     - name: Build Website
-  #       run: |
-  #         docker run -v ${PWD}/docs/sources:/hugo/content/docs/loki/latest --rm grafana/docs-base:latest /bin/bash -c 'mkdir -p content/docs/grafana/latest/ && touch content/docs/grafana/latest/menu.yaml && make prod'
-  sync:
-    runs-on: ubuntu-latest
-    # needs: test
-    steps:
-    - uses: actions/checkout@v3
-    - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
-    - name: publish-to-git
-      uses: ./.github/actions/website-sync
-      id: publish
-      with:
-        repository: grafana/website
-        branch: master
-        host: github.com
-        github_pat: '${{ secrets.GH_BOT_ACCESS_TOKEN }}'
-        source_folder: docs/sources
-        target_folder: content/docs/loki/next
-    - shell: bash
-      run: |
-        test -n "${{ steps.publish.outputs.commit_hash }}"
-        test -n "${{ steps.publish.outputs.working_directory }}"
```

Comment:
Not sure what this step does 🤔 I assume it is simply to test if the previous step was successful.
But it is not included in the new actions.
Is it ok to remove it?